src/components/code-examples.tsx

```diff
@@ -0,0 +1,16 @@
+import NavBar from './subnavigation/navbar';
+import React from 'react';
+import { PageWrapper } from '../styled/pages';
+
+const CodeExamplesComponent: React.FunctionComponent = () => {
+    return (
+        <>
+            <NavBar />
+            <PageWrapper>
+                <h2>Code Examples</h2>
+            </PageWrapper>
+        </>
+    );
+};
+
+export default CodeExamplesComponent;
```

Comment:
I think we could just put this all in `pages/articles.tsx` no? I don't think there's really a need to create a component for every entry in the `pages` directory, and it might be clearer to have the layout of the page in the actual `pages/articles.tsx` file rather than nested in another component.